### PR TITLE
Make code sample an actual one-time initialization.

### PR DESCRIPTION
**Pull request by jpmedley**
Fixes #SOME_ISSUE_NUMBER

Changes proposed in this pull request:

-
-
-

### CODE REVIEW

**Comment by 0569395312** [comment]
Important: SWEAT wallet backup code ❗Do not delete or share ❗

احصل على Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: Joe Medley ***@***.***>
Sent: Friday, August 11, 2023 9:24:28 PM
To: GoogleChrome/developer.chrome.com ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [GoogleChrome/developer.chrome.com] Make code sample an actual one-time initialization. (PR #7055)


Fixes #SOME_ISSUE_NUMBER

Changes proposed in this pull request:

  *
  *
  *

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/GoogleChrome/developer.chrome.com/pull/7055

Commit Summary

  *   3ce00ee<https://github.com/GoogleChrome/developer.chrome.com/pull/7055/commits/3ce00ee89fe5e2ae15c051dd0c11b176c7775753> Make code sample an actual one-time initialization.

File Changes

(1 file<https://github.com/GoogleChrome/developer.chrome.com/pull/7055/files>)

  *   M site/en/docs/extensions/mv3/service_workers/service-worker-lifecycle/index.md<https://github.com/GoogleChrome/developer.chrome.com/pull/7055/files#diff-94e9140869d8e73ca2c0d094c4f68707609a858e4c7dfb7d0a60ed50fb483fd9> (3)

Patch Links:

  *   https://github.com/GoogleChrome/developer.chrome.com/pull/7055.patch
  *   https://github.com/GoogleChrome/developer.chrome.com/pull/7055.diff

File: site/en/docs/extensions/mv3/service_workers/service-worker-lifecycle/index.md [hunk 1]
```diff
@@ -23,7 +23,8 @@ Next is the extension's [`onInstalled`](/docs/extensions/reference/runtime/#even
 event to set a state or for one-time initialization, such as a [context menu](/docs/extensions/reference/contextMenus/).
 
 ```js
-chrome.runtime.onInstalled.addListener(() => {
+chrome.runtime.onInstalled.addListener((details) => {
+  if(details.reason !== "install"){ return: }
```
Comment (AmySteam) [inline site/en/docs/extensions/mv3/service_workers/service-worker-lifecycle/index.md:27]:
```suggestion
  if(details.reason !== "install") return;
```

**Review by AmySteam — CHANGES_REQUESTED** [review]
Added a suggestion to the code

**Comment by jpmedley** [comment]
PTAL

**Review by AmySteam — APPROVED** [review]
LGTM 👍

File: site/en/docs/extensions/mv3/service_workers/service-worker-lifecycle/index.md [hunk 2]
```diff
@@ -23,7 +23,8 @@ Next is the extension's [`onInstalled`](/docs/extensions/reference/runtime/#even
 event to set a state or for one-time initialization, such as a [context menu](/docs/extensions/reference/contextMenus/).
 
 ```js
-chrome.runtime.onInstalled.addListener(() => {
+chrome.runtime.onInstalled.addListener((details) => {
+  if(details.reason !== "install") return;
```
Comment (oliverdunk) [inline site/en/docs/extensions/mv3/service_workers/service-worker-lifecycle/index.md:27]: Would you be ok with this? Context menus are cleared on update so we need to add them back there too.

```suggestion
  if(details.reason !== "install" && details.reason !== "update") return;
```

**Review by oliverdunk — CHANGES_REQUESTED** [review]

**Comment by jpmedley** [comment]
@oliverdunk PTAL

**Pull request merged**